packages/bpk-component-image/src/withLazyLoading.js

```diff
@@ -36,9 +38,11 @@ export default function withLazyLoading(Component, document) {
     }
 
     componentDidMount() {
-      const passiveArgs = this.supportsPassiveEvents() ? { passive: true } : false;
-      document.addEventListener('scroll', this.checkInView, passiveArgs);
+      const passiveArgs = this.supportsPassiveEvents() ? { passive: true } : {};
+      this.scrollEventArgs = { capture: true, ...passiveArgs };
```

Comment:
Hmm we should have a test for this behaviour.

I.e. a test that asserts that `this.scrollEventArgs` is the same after `componentDidMount ` and during `componentWillUnmount`.

Reply:
I feel like this is just testing javascript?? Maybe worthwhile in-case someone changes the args?

Reply:
Actually, come to think of it, we don't want to test that they are the same - as this is internal workings of the HOC. What we want to test is that the event listeners are, indeed, added and removed correctly

Reply:
IMHO